src/graph/validator/Validator.cpp

```diff
@@ -299,8 +302,14 @@ std::vector<std::string> Validator::getOutColNames() const {
 Status Validator::appendPlan(PlanNode* node, PlanNode* appended) {
   DCHECK(node != nullptr);
   DCHECK(appended != nullptr);
-  if (!node->isSingleInput()) {
-    return Status::SemanticError("%s not support to append an input.",
+
+  std::set<PlanNode::Kind> skipTypeSet = {PlanNode::Kind::kSwitchSpace};
+
+  // Note: there's check which forbids sentence like `[dql(or other) ngql];[cypher]`.
```

Comment:
This check don't report error?

Reply:
> This check don't report error?

like `use nba; match (n:player) return n limit 1`, it's right

Reply:
But we have disable mixed ngql and cypher as you said in this comment.